src/markdown-pages/automate-workflows/get-started-kubernetes.mdx

```diff
@@ -204,19 +204,37 @@ To finish things off, you'll create and add an alert channel to your alert polic
 
 ## Try it out now
 
-We have a Kubernetes test cluster ready for you in 2 minutes.  By following this on-line tutorial, you will learn how to:
+We have a Kubernetes test cluster ready for you in 2 minutes. By following this on-line tutorial, you will learn how to:
+
 - Deploy the New Relic agent in a Kubernetes environment
 - Use the New Relic Kubernetes operator
 
 **Some tips to use the on-line tutorial window:**
+
 - Accept the cookies, so you can see the menu bar.
 - Click anywhere in the tutorial window to start. It will take about 2 minutes for your environment to be ready.
 - Press CTRL-l or type `clear` to clear the terminal window
 - Click on the finish flag icon in the bottom menu to hide or show the instructions
 
 Good luck!
 
-<iframe width="1000" height="1000" sandbox="allow-same-origin allow-scripts allow-popups allow-forms" src="https://play.instruqt.com/embed/newrelic/nr-k8s-operator" style="border: 0;"></iframe>
+<Important>
+
+Some browsers automatically disable the use of iframes. If the module isn't loading
+please check your browser settings.
+
+</Important>
+
+<iframe
+  title="Instruct K8 Operator Module"
+  width="1000"
+  height="1000"
+  sandbox="allow-same-origin allow-scripts allow-popups allow-forms"
+  src="https://play.instruqt.com/embed/newrelic/nr-k8s-operator"
+  style="border: 0;"
+>
+  <p>Your browser does not support iframes.</p>
```

Comment:
Out of curiosity, have you tried putting the `<Important>` tag _inside_ the `iframe` tag? I suspect it wouldn't work, but I could see a benefit in only showing the alert for the impacted users, not everyone.

Reply:
I have not, but can test it out!

Reply:
It didn't help, so we can commit this as a temp solution.